HomeWorks/Homework12Nbrb/TMS.Homework.Nbrb/FileLibrary/File.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace FileLibrary
+{
+    public class File
```

Comment:
Please do not use names of existing classes, it can confuse no only people but sometimes compiler